cmake/Dependencies.cmake

```diff
@@ -1150,6 +1150,22 @@ if(USE_CUDA)
   include_directories(SYSTEM ${CUB_INCLUDE_DIRS})
 endif()
 
+# ---[ XCCL
+if(USE_XCCL)
+  if(NOT USE_XPU)
+    message(WARNING
+        "Not using XPU, so disabling USE_NUSE_XCCLCCL. Suppress this warning with "
```

Comment:
Hi, here's a typo `USE_NUSE_XCCLCCL` I think, the name is `USE_XCCL` in warning message?

Reply:
Thank you for your Comments, and I apologize for letting a draft appear in the official PR. There were many typos that I didn't tidy up in time. I will complete the overall work and submit the PR again. Thanks again!